json_schema/core/file/file_core.json

```diff
@@ -1,76 +1,76 @@
 {
-    "$schema": "http://json-schema.org/draft-07/schema#", 
-    "description": "Information about a file produced from any process.", 
-    "additionalProperties": false, 
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "description": "Information about a file produced from any process.",
+    "additionalProperties": false,
     "required": [
-        "file_name", 
+        "file_name",
         "format"
-    ], 
-    "title": "File core", 
-    "name": "file_core", 
-    "type": "object", 
+    ],
+    "title": "File core",
+    "name": "file_core",
+    "type": "object",
     "properties": {
         "describedBy": {
-            "description": "The URL reference to the schema.", 
-            "type": "string", 
+            "description": "The URL reference to the schema.",
+            "type": "string",
             "pattern": "https://raw.githubusercontent.com/morphic-bio/metadata-schema/main/json_schema/core/file/1.0.0/file_core"
```

Comment:
We could use [constant](https://json-schema.org/understanding-json-schema/reference/generic.html#constant-values) for this instead of `pattern`

Reply:
Good idea. I will check it out.

Reply:
I will create a test case demonstrating your changes with pointing to an older version.